test_cases/001/petab/mapping_table.tsv

```diff
@@ -1,4 +1,4 @@
-netId	ioId	ioValue
-net1	input1	prey
-net1	input2	predator
-net1	output1	gamma
+petab.MODEL_ENTITY_ID	petab.PETAB_ENTITY_ID
```

Comment:
typos

Reply:
Maybe I need new glasses, but what is the typo?

Reply:
Ohh following #4 I thought this was the syntax..., but thanks for noticing I will change!

Reply:
Next time I should read the spec more carefully, but now fixed :)